docs/development/release.md

```diff
@@ -55,24 +64,42 @@ Where `${release_version}` corresponds to the version tag `v1.2.3`, for example.
 
 Note: This is for 1.10.3 or later.
 
-### 6. Create a pull request against the master branch
+### 6. Create a pull request against the feature branch
```

Comment:
Maybe change `feature` to `minor version`?

Reply:
Hmm, sure we can change that wholesale

Reply:
db4c8d4